### PR TITLE
Fixed cross-site scripting via svg file upload

### PR DESCRIPTION
**Pull request by girishpanchal30**
### Summary

In this PR I've filtered the SVG image content and removed the harmful code to fix the cross-site scripting vulnerability issue.

## Check before Pull Request is ready:

* [ ] I have [written a test](CONTRIBUTING.md#writing-an-acceptance-test) and included it in this PR
* [x] I have [run all tests](CONTRIBUTING.md#run-tests) and they pass
* [x] The code passes when [running the PHP CodeSniffer](CONTRIBUTING.md#run-php-codesniffer)
* [x] Code meets [WordPress Coding Standards](CONTRIBUTING.md#coding-standards) for PHP, HTML, CSS and JS
* [x] [Security and Sanitization](CONTRIBUTING.md#security-and-sanitization) requirements have been followed
* [x] I have assigned a reviewer or two to review this PR (if you're not sure who to assign, we can do this step for you)

Closes https://github.com/Codeinwp/neve-pro-addon#2842

### CODE REVIEW

File: obfx_modules/custom-fonts/custom_fonts_admin.php [hunk 1]
```diff
@@ -404,4 +404,39 @@ public function update_mime_types( $defaults, $file, $filename ) {
...
+		}
+		// Get file type.
+		$filetype = wp_check_filetype( $file['name'], null );
+		// Check is svg image.
+		if ( 'image/svg+xml' === $filetype['type'] ) {
+			$contents = $wp_filesystem->get_contents( $file['tmp_name'] );
+			// Remove potentially harmful code from SVG content.
+			$contents = preg_replace( '/<script\b[^>]*>(.*?)<\/script>/is', '', $contents );
+			$contents = preg_replace( '/<!ENTITY(.*?)>/is', '', $contents );
+			$contents = preg_replace( '/<!DOCTYPE(.*?)>/is', '', $contents );
```
Comment (Soare-Robert-Daniel) [inline obfx_modules/custom-fonts/custom_fonts_admin.php:433]: Just a small suggestion. We can combine them in a single call:

```php
$contents = preg_replace(
	[
		'/<script\b[^>]*>(.*?)<\/script>/is',
		'/<!ENTITY(.*?)>/is',
		'/<!DOCTYPE(.*?)>/is'
	], 
	'',
	$contents
);
```

**Review by Soare-Robert-Daniel — APPROVED** [review]

File: obfx_modules/custom-fonts/custom_fonts_admin.php (hunk 1 above)
Comment (girishpanchal30) [reply obfx_modules/custom-fonts/custom_fonts_admin.php]: @Soare-Robert-Daniel Done! Thank you for the review

**Comment by selul** [comment]
@girishpanchal30 please use this library instead for sanitization https://github.com/Codeinwp/optimole-wp/blob/master/composer.json#L59, you can check Optimole on how is used.

**Comment by girishpanchal30** [comment]
@selul Thank you for the suggestion, I've Implemented the library with the latest commit.

**Pull request merged**